lib/rules/execute-rule.js

```diff
@@ -1,37 +1,30 @@
-'use strict';
+const isString = require('lodash.isstring');
+const getInputFromRule = require('./get-input-from-rule');
+const executeRuleList = require('./execute-rule-list');
+const executeRuleEqual = require('./execute-rule-equal');
+const executeRulePercent = require('./execute-rule-percent');
 
-var _ = require('lodash');
-var getInputFromRule = require('./get-input-from-rule');
-var executeRuleList = require('./execute-rule-list');
-var executeRuleEqual = require('./execute-rule-equal');
-var executeRulePercent = require('./execute-rule-percent');
-
-module.exports = executeRule;
-
-//We assume parameters are validated before they reach this point.
-function executeRule(input, rule, values, cb) {
-  var extractedInput = getInputFromRule(input, rule);
-  if(!_.isString(extractedInput))
-    //We can't execute rule if input was not found,
-    //or it is not a string.
-    //The inputValidator should be used to catch
-    //situations when the input passed in is invalid.
-    //If the input passes validation, we assume that a
-    //missing input should skip rule execution and either
-    //return the default value for that key, or undefined
-    //if no default value is set.
-    return void cb();
-  switch(rule.type) {
+// We assume parameters are validated before they reach this point.
```

Comment:
Maybe remove `We` tense to be more neutral. Same goes for other comments in this file. 
`Parameters are assumed to be validated before they reach this point.`